src/giftCards/GiftCardSettings/GiftCardExpirySettingsCard/GiftCardExpirySettingsCard.tsx

```diff
@@ -16,17 +17,30 @@ export interface GiftCardExpirySettingsCardProps
   onChange: (event: React.ChangeEvent<any>) => void;
 }
 
+const useStyles = makeStyles(
+  () => ({
+    cardTitle: {
+      paddingTop: 0,
+    },
+  }),
+  { name: "GiftCardExpirySettingsCard" },
+);
+
 const GiftCardExpirySettingsCard: React.FC<GiftCardExpirySettingsCardProps> = ({
   data,
   disabled,
   errors,
   onChange,
 }) => {
   const intl = useIntl();
+  const classes = useStyles({});
 
   return (
     <Card data-test-id="gift-card-settings">
-      <CardTitle title={intl.formatMessage(messages.expiryDateTitle)} />
+      <CardTitle
+        title={intl.formatMessage(messages.expiryDateTitle)}
+        className={classes.cardTitle}
```

Comment:
question: why not use here sprinkles with 0 padding ?

Reply:
Too low specificity:
![CleanShot 2023-02-14 at 15 37 06@2x](https://user-images.githubusercontent.com/9116238/218769369-c0111421-75bc-4566-ad99-cb68a1012391.jpg)